src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -389,6 +393,26 @@ class IpexCommunicationService extends AgentService {
     });
   }
 
+  @OnlineOnly
+  async handleMessageFromMultisigExn(id: string) {
+    const notifRecord = await this.getNotificationRecordById(id);
+    const exn = await this.props.signifyClient
+      .exchanges()
+      .get(notifRecord.a.d as string);
+    const route = exn?.exn?.e?.exn?.r;
+
+    switch (route) {
+    case ExchangeRoute.IpexAdmit:
+      await this.acceptAcdcFromMultisigExn(id);
+      break;
+    case ExchangeRoute.IpexGrant:
+      await this.acceptPresentAcdcFromMultisigExn(id);
```

Comment:
this also probably conflicts, because we might just be receiving an issuance from the server to a multi-sig identifier and it will call this